_api_app/app/Http/routes.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Application Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register all of the routes for an application.
+| It is a breeze. Simply tell Lumen the URIs it should respond to
+| and give it the Closure to call when that URI is requested.
+|
+*/
+
+// $app->get('/', function () use ($app) {
+//     return 'Nothing here. Go away!';
+// });
+
+$app->group(['prefix' => 'v1','namespace' => 'App\Http\Controllers'], function($app) {
+	$app->get('state','StateController@getState');
+});
```

Comment:
new line at the end of file
most editors, have that feature or plugin that adds new line at the end of file if it isn't already there